### ISSUE TITLE
Only part of the waveform is rendered when the browser window isn't maximized

### ISSUE THREAD

**Issue by FraserThompson** [katspaugh/wavesurfer.js#1175]
### Wavesurfer.js version(s):

Every version since 1.3.0 up until 2.0.0 beta. The behaviour isn't there in 1.2.8.

### Browser version(s):

Chrome 60, Edge, Internet Explorer 11, Firefox

### Code needed to reproduce the issue:
JS:
```
    this.wavesurfer = WaveSurfer.create({
      container: this.waveform,
      waveColor: color,
      height: height,
      hideScrollbar: true,
      normalize: true,
      pixelRatio: "1",
      forceDecode: false,
      backend: "MediaElement",
      progressColor: 'orange',
      barWidth: '2'
    });
```

CSS:
```
.waveform {
	opacity: 1;
	transition: opacity 250ms ease-in-out;
	width: 100%;
	position: relative;
}

#waveform_wrapper {
	height: 60px;
	opacity: 1;
	width: 100%
}
```

HTML:
```
<div id="waveform_wrapper" class="player row" data-artist="{{include.artist}}"">
    <div class="col-xs-12" style="padding-top: 5px;">
        <div class="length-wrapper" style="left: 45px;">
            <span class="currentTime"></span>
        </div>
        <div class="waveform" id="waveform_{{include.machine_name}}"></div>
        <div class="length-wrapper" style="right: 0px;">
            <span class="length"></span>
        </div>
    </div>
</div>
```

### Use behaviour needed to reproduce the issue:

Loading an MP3 and Waveform into Wavesurfer when the browser window isn't maximised (or the developer console is open) will cause it to only render part of the waveform, like this:

![image](https://user-images.githubusercontent.com/6093858/28487190-c80e1f78-6edf-11e7-9f2e-5ecdf605db57.png)

If you maximise the window and then load the waveform it renders fine.

When I inspect I can see the canvas is there and is the correct width, it's just only being partially displayed.

**Comment by entonbiba** [comment]
@FraserThompson  this has already been fixed with pr #1114 but it hasn't been merged yet.

**Comment by FraserThompson** [comment]
Thanks. Is there a workaround I can use until that's merged? What's the actual cause?

**Comment by matixmatix** [comment]
@entonbiba looking forward to that being merged, is there an estimation when it will happen?

**Comment by entonbiba** [comment]
@matixmatix not sure, hopefully soon. I added the fix to version 1.4.0 and attached it here:
[waveform-fix.zip](https://github.com/katspaugh/wavesurfer.js/files/1192038/waveform-fix.zip)

**Comment by mspae** [comment]
Duplicate of #1182
